nmmo/task/predicate.py

```diff
@@ -0,0 +1,256 @@
+from typing import Dict, List, Tuple
+
+from nmmo.task.game_state import GameState
+
+
+# TODO(kywch): Revisit Team and re-evaluate if putting it here makes sense
+# NOTE: changed Team to Group, to be more general
+class Group:
+  def __init__(self, agents:List[int], name:str=None):
+    assert len(agents) > 0, "Team must have at least one agent"
+    self.name = name if name else f"Agent{str(agents).replace(' ', '')}"
```

Comment:
How about f"Agents("{",".join(agents)}")"

Reply:
Changed to `f"Agent({','.join([str(e) for e in agents])})"`, which produces something like: 'Agent(4,5,6,7)'

It seems that we also would a group of one agent a lot, so I'm leaning toward Agent (vs. Agents)